README.md

```diff
@@ -87,7 +87,7 @@ git clone git@github.com:CMSgov/beneficiary-fhir-data.git ~/workspaces/bfd/benef
 </toolchains>
 ```
 4. Change to the `apps/` directory and `mvn clean install -DskipITs`. The flag to skip the integration tests is important here. You will need to have AWS access for the integration tests to work correctly.
-5. Set up a Postgres 11 database. The easiest way to set up a local database is with the following command. Data will be persisted between starts and stops in the `bfd_pgdata` volume.
+5. Set up a Postgres 12 database. The easiest way to set up a local database is with the following command. Data will be persisted between starts and stops in the `bfd_pgdata` volume.
```

Comment:
I coincidentally made this change in my PR for the postgres docker container, but I also added more in this area to explain running the automated script. LGTM. Ill resolve the merge conflict.